### PR TITLE
Algod: rename crypto-stateproof-builder to crypto-stateproof-prover

### PR DESCRIPTION
**Pull request by algoidan**
## Summary
This change should reduce the confusion around the  term "builder" that is being used for the 
state proof crpyto code, and the state proof go-routine that is responsible for orchestrating the state proof creation in the network

### CODE REVIEW

File: stateproof/worker.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ import (
 type builder struct {
```
Comment (bbroder-algo) [inline stateproof/worker.go:39]: what about here

Comment (bbroder-algo) [reply stateproof/worker.go]: oh sorry!  missed the draft part.  duh.

File: stateproof/db.go [hunk 2]
```diff
@@ -47,17 +47,17 @@ const (
 
 	createSigsIdx = `CREATE INDEX IF NOT EXISTS sigs_from_this_node ON sigs (from_this_node)`
 
-	// builders table stored a serialization of each BuilderForRound data, without the sigs (stored separately)
-	createBuildersTable = `CREATE TABLE IF NOT EXISTS builders (
+	// provers table stored a serialization of each ProverForRound data, without the sigs (stored separately)
+	createProverTable = `CREATE TABLE IF NOT EXISTS builders (
```
Comment (algoidan) [inline stateproof/db.go:51]: I'm in favor of changing the table name as well.

**Review by algorandskiy — COMMENTED** [review]
looks all right. please remove the sqlite files and rename the table

**Review by algorandskiy — APPROVED** [review]

File: stateproof/db.go [hunk 3]
```diff
@@ -47,17 +47,17 @@ const (
...
-	createBuildersTable = `CREATE TABLE IF NOT EXISTS builders (
+	// provers table stored a serialization of each ProverForRound data, without the sigs (stored separately)
+	createProverTable = `CREATE TABLE IF NOT EXISTS provers (
     	round INTEGER PRIMARY KEY NOT NULL,
-    	builder BLOB NOT NULL
+    	prover BLOB NOT NULL
     )`
 
-	insertOrReplaceBuilderForRound = `INSERT OR REPLACE INTO builders (round,builder) VALUES (?,?)`
+	insertOrReplaceProverForRound = `INSERT OR REPLACE INTO provers (round,prover) VALUES (?,?)`
```
Comment (algonautshant) [inline stateproof/db.go:56]: Will this cause any trouble on Alphanet?

**Review by algonautshant — APPROVED** [review]
Indeed, just a renaming, and a very welcomed one :-) 
Since the db name is changing, will this cause any problem with Alphanet? 
Perhaps not, but the old db may remain there forever.

**Pull request merged**